modules/.gitignore

```diff
@@ -0,0 +1,2 @@
+*/bin
+*/obj
```

Comment:
Consider adding more of the `.gitignore` essentials here, as it will override all global ignores.

Reply:
This doesn't seem to be an issue for me